app/javascript/packs/application.js

```diff
@@ -3,7 +3,9 @@ import main from '../mastodon/main';
 if (!window.Intl || !Object.assign || !Number.isNaN ||
     !window.Symbol || !Array.prototype.includes) {
   // load polyfills dynamically
-  import('../mastodon/polyfills').then(main);
+  import('../mastodon/polyfills').then(main).catch(e => {
+    // console.log(e); // no-op
+  });
```

Comment:
However console logs are not allowed in Mastodon - it fails the build. I can't think of another way to get around that and still have this catch be useful. I think it's sort of a hint for debugging. Can you think of another way to get around the fact that Mastodon does not allow console logging?